app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ActiveRecord::Base
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  has_many :likes, class_name: 'Like', foreign_key: :post_id
+  has_many :comments, foreign_key: :post_id
+
+  def update_user_posts_counter
+    author.increment!(posts_counter)
+  end
+
+  def recent_comment
+    comments.limit(5).order(created_at: :asc)
+  end
+end
```

Comment:
It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save `to call the #update_user_posts_counter method every time a new post has been saved to database, then make the update_user_posts_counter method private 👍🏽